main.go

```diff
@@ -41,20 +41,21 @@ func main() {
 
 	for _, singlePackage := range packages {
 		vulnerabilitiesOSS, err := ossFetcher.Test(singlePackage.Name, singlePackage.Version)
-		//		vulnerabilitiesNodeSWG, err := nodeswgFetcher.Test()
+		// vulnerabilitiesNodeSWG, err := nodeswgFetcher.Test(singlePackage.Name, singlePackage.Version)
```

Comment:
Why are you replacing commented code with other commented code?

Reply:
@Elexy I should have removed it : when I saw it was commented, I un-commented it to understand what it did, then I saw it was not the actual API and fixed it, then I saw it was no useful as is for now.
Though it remembered be it should be done in parallel rather than the actual sequential way (see the whole function code) so I kept it, more as a reminder, but I should drop it :+1: